lib/ProductOpener/Display.pm

```diff
@@ -7193,6 +7201,10 @@ HTML
 	my $html;
 
 	# init javascript code
+  # Kept those 3 lines during the rebase. Not evident if needed @CharlesNepote
```

Comment:
@CharlesNepote I rebased the Power User Script branch. but I was not sure about those 3 lines. Can I delete them ?